sycl/include/CL/sycl/detail/spirv.hpp

```diff
@@ -216,24 +216,25 @@ EnableIfGenericBroadcast<T> GroupBroadcast(T x, id<Dimensions> local_id) {
 
 // Single happens-before means semantics should always apply to all spaces
 // Although consume is unsupported, forwarding to acquire is valid
+template <typename T>
 static inline constexpr __spv::MemorySemanticsMask::Flag
```

Comment:
It would be useful to have SFINAE check here that T is equal to either ONEAPI::memory_order or sycl::memory_order